src/Configuration/ECSConfigBuilder.php

```diff
@@ -73,8 +86,12 @@ final class ECSConfigBuilder
 
     private ?bool $reportingRealPath = null;
 
+    private bool $useEditorConfig = false;
```

Comment:
this should be nullable bool like other, to avoid import multiple config via EcsConfigBuilder which cause it to be false again.

```suggestion
    private ?bool $useEditorConfig = null;
```

Reply:
Will update, thanks :+1: